databases/week3/additional_queries.sql

```diff
@@ -0,0 +1,71 @@
+/*Additional queries*/
+/*Get meals that has a price smaller than a specific price fx 90*/
+SELECT
+  *
+FROM
+  meal
+WHERE
+  price < 120
+LIMIT
+  100;
```

Comment:
Remove lines 9 & 10 , why we should limit to 100 meals...? we should test all meals if they fulfill above condition